logstreams/src/main/java/io/zeebe/logstreams/processor/StreamProcessorController.java

```diff
@@ -268,6 +268,8 @@ protected void onActorClosing() {
           onCommitPositionUpdatedCondition);
       onCommitPositionUpdatedCondition = null;
     }
+
+    LOG.debug("Closed stream processor controller {}.", getName());
```

Comment:
I think this log statement could be printed before the stream processor is really closed ;)

Reply:
done